recipes/grpc/all/conanfile.py

```diff
@@ -101,6 +101,8 @@ def requirements(self):
         self.requires("protobuf/3.21.12", transitive_headers=True, transitive_libs=True)
         self.requires("re2/20230301")
         self.requires("zlib/[>=1.2.11 <2]")
+        if self.settings.os in ["Linux", "FreeBSD"] and Version(self.version) >= "1.52":
```

Comment:
Maybe it would make sense to move this condition to new method? If there would be some change in this condition in future, it might be easy to miss second usage.

Reply:
You are right but let's avoid recompiling everything right now. We'll be happy to change this when more consequential changes are needed.